test-suite-caffeine-native/src/test/java/io/micronaut/cache/NewsControllerTest.java

```diff
@@ -0,0 +1,40 @@
+package io.micronaut.cache;
+
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.http.uri.UriBuilder;
+import io.micronaut.runtime.server.EmbeddedServer;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+
+import java.time.Month;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@MicronautTest
+class NewsControllerTest {
+
+    @Inject
+    EmbeddedServer server;
```

Comment:
It seems that `server` is not used in this class so not sure why it needs to be injected?

Reply:
You're right, I copied it blindly from https://github.com/micronaut-graal-tests/micronaut-cache-graal/blob/3.10.x/src/test/java/example/micronaut/NewsControllerTest.java#L21-L22

Gone now 😄👍